examples/examples.py

```diff
@@ -51,22 +52,27 @@
         "scenario": "example_01d",
         "study_case": "zonal_case",
     },
-    "large_2019_eom": {"scenario": "example_02", "study_case": "base_case_2019"},
+    # example_01e is used in the tutorial notebook #3: Custom unit and bidding strategy example
+    # example_01f is used in the tutorial notebook #5: Market configuration comparison example
```

Comment:
Why are they not here anymore?

Reply:
they were never here in the dictionary. I have just added these comments so if someone sees those files and doesn't see them in this dict, they know what they are